src/meta/play-meta.js

```diff
@@ -58,6 +59,17 @@ const plays = [
     level: 'Intermediate',
     tags: 'Recursion, Tree',
     github: 'green-roots'
+  }, {
+    id: 'pl-counter', 
+    name: 'Counter', 
+    description: 'A simple counter',
+    component: () => {return <CounterApp />},
+    path: '/plays/Counter',
+    level: 'Beginner',
+    tags: 'JSX, State, Props',
+    github: 'murtuzaalisurti',
+    blog: 'https://syntackle.vercel.app',
```

Comment:
Do we have any direct blog article link that explains building this play?

Reply:
Ohh, so we have to add a tutorial, right? I thought we just had to add a blog, 😄 
But, right now there is no tutorial for this demo so I have removed the link!

Reply:
No worries. We can always update it if you have an article on it in the future.

Btw, our user profiles like image, website, blog all will be fetched using the Github user id and will be shown on the play page.